avro-data/src/test/java/io/confluent/connect/avro/AvroDataTest.java

```diff
@@ -503,22 +490,35 @@ public void testFromConnectComplexWithDefaults() {
     org.apache.avro.Schema avroSchema = org.apache.avro.SchemaBuilder
         .record(AvroData.DEFAULT_SCHEMA_NAME).namespace(AvroData.NAMESPACE) // default values
         .fields()
-        .name("int8").doc("int8 field").type(int8Schema).withDefault(2)
-        .name("int16").doc("int16 field").type(int16Schema).withDefault(12)
-        .name("int32").doc("int32 field").type(int32Schema).withDefault(12)
-        .name("int64").doc("int64 field").type(int64Schema).withDefault(12L)
-        .name("float32").doc("float32 field").type(float32Schema).withDefault(12.2f)
-        .name("float64").doc("float64 field").type(float64Schema).withDefault(12.2)
-        .name("boolean").doc("bool field").type(boolSchema).withDefault(true)
-        .name("string").doc("string field").type(stringSchema).withDefault("foo")
-        .name("bytes").doc("bytes field").type(bytesSchema).withDefault(ByteBuffer.wrap("foo".getBytes()))
+        .name("int8").type(int8Schema).withDefault(2)
+        .name("int16").type(int16Schema).withDefault(12)
+        .name("int32").type(int32Schema).withDefault(12)
+        .name("int64").type(int64Schema).withDefault(12L)
+        .name("float32").type(float32Schema).withDefault(12.2f)
+        .name("float64").type(float64Schema).withDefault(12.2)
+        .name("boolean").type(boolSchema).withDefault(true)
+        .name("string").type(stringSchema).withDefault("foo")
+        .name("bytes").type(bytesSchema).withDefault(ByteBuffer.wrap("foo".getBytes()))
         .name("array").type(arraySchema).withDefault(Arrays.asList("a", "b", "c"))
```

Comment:
I think the `doc` calls in test are in line 363-372.  
`avroSchema` is expected schema, and it will have properties "connect.field.doc.*" See line 509 - 521

Reply:
avroSchema (what we expect) will not have field doc because we will no longer add content of `connect.doc` to avro field's doc, it will have the `connect.doc` property at field schema level instead.